src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-label="selectLabel"
+      :aria-owns="id"
+      :aria-required="isRequired"
+      :aria-describedby="'error' + id"
```

Comment:
Wouldn't we want this to be conditional if there actually is an error?

Reply:
im not sure, the span is hidden unless the error surfaces, i don't know enough about ally, thoughts @eleanormal?

Reply:
If the span is empty nothing will be read